### DISCUSSION TITLE
General Discussion

### DISCUSSION THREAD

**Discussion by manishawsdevops** [manishawsdevops/aws-devops-learning#15] [General]
Please make use of this discussions for any discussions.

**Comment by SaiPrakash0630** [comment]
Assignment 2 task 6: I have ssh for server2, it didn't work, changed the authorized keys too.

**Comment by SaiPrakash0630** [comment]
Hi, I have committed and done a pull request. Please confirm did you get the pull request?

**Comment by SaiPrakash0630** [comment]
I have committed ec2 with AMI task. You can review it.

**Comment by SaiPrakash0630** [comment]
I have committed assignment-5 in a new branch. I have committed using .md file in AutoScaling Assignment folder under EC2 directory.

**Comment by SaiPrakash0630** [comment]
I have committed VPC task. Please once confirm when you got it.
